templates/microsoft_cloud_for_industry/modules/policy_remediation/main.tf

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/*
+SUMMARY : Common module for creating policy remediations
+AUTHOR/S: Cloud for Industry
+*/
+resource "azurerm_management_group_policy_remediation" "policy_remediation" {
```

Comment:
Per @matt-FFFFFF previous discussion, these remediations will have no effect and should be removed from the module.